celi_framework/main.py

```diff
@@ -146,6 +227,20 @@ def instantiate_with_argparse_args(args: argparse.Namespace, cls: Type):
 
 
 if __name__ == "__main__":
+    tool_config_path = get_default_config_path()
```

Comment:
This line of code and the one below it don't do anything.  In fact, they may print a misleading message.
tool_config_path isn't used after the print statement.